File: ASP.Net Core MVC/RazorPageApp/Models/Student.cs
using System.ComponentModel.DataAnnotations;

namespace RazorPageApp.Models
{
    public class Student
    {
        public int Id { get; set; }
        [Required]
        public string Name { get; set; }
        [Range(10,20)]
        public int Age { get; set; }
    }
}

File: Angular/Labs/Lab3/src/app/_models/student.ts
// export class Student {
//   id : number;
//   name : string;
//   age : number;
//   deptNo : number;
//   constructor (id:number = 0,name:string = "",age:number = 15,deptNo:number = 4) {
//     this.id = id;
//     this.name = name;
//     this.age = age;
//     this.deptNo = deptNo;
//   }
// }

export class Student {

  constructor (public id:number = 0,public name:string = "",public age:number = 15,public deptNo:number = 4) {

  }
}